astroquery/noirlab/core.py

```diff
@@ -25,7 +25,10 @@ class NoirlabClass(BaseQuery):
     def __init__(self, which='file'):
         """Return object used for searching the NOIRLab Archive.
 
-        Search either for Files (which=file) or HDUs (which=hdu)."""
+        Search either Files (which=file) or HDUs (which=hdu).
+        Files will always be returned.  But if which=hdu, 
+        the 
```

Comment:
Looks like you committed this halfway done?

Reply:
Good catch, fixed.